test/cases/testdata/numbersrangestep/test-numbersrangestep-1.yaml

```diff
@@ -0,0 +1,20 @@
+cases:
+- data: {}
+  modules:
+  - |
+    package generated
+
+    p = __local0__ {
```

Comment:
I presume you've had a look at other examples for these tests, so I don't blame you for copying their style — but many of them were generated from code eons ago :) There's no need to use var names like `__local0__` here. Using normal variable names and identifiers is preferable :)

Also, using a more descriptive package name than `generated` would be good.

Reply:
Thank you for the explanation! I was confused why `__local0__` was being used. I've updated the tests as you suggested and to follow [the style outlined by the documentation instead](https://www.openpolicyagent.org/docs/latest/contrib-adding-builtin-functions/#test).